src/style.css

```diff
@@ -0,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  --border_radius: 4px;
+}
+
+.big_search_bar,
+.small_search_bar {
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: var(--border_radius);
+  margin: 20px 0;
+}
+
+.icon_big_search_bar {
+  position: absolute;
+  padding: 25px 17px 26px 26px;
+}
+
+.icon_small_search_bar {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  padding: 15px 9px 16px 13px;
+}
+
+.input_small_bar,
+.input_big_bar {
```

Comment:
You may also use two classes - one which specifies common styles for both big and small inputs, and two other, that specifies exact styles for two inputs separately (e.g:
`class="searchInput searchInput_small"
class="searchInput searchInput_large"`